src/common/settings.ts

```diff
@@ -122,4 +127,18 @@ document.head.appendChild(stylesheetLink);
 
 ReactDOM.createRoot(document.getElementById("root") as HTMLElement);
 
+//Update the appIDs in apple-app-site-association.json
+const templatePath = "/.well-known/apple-app-site-association";
+
+fetch(templatePath)
```

Comment:
better to do this in https://github.com/Tonomy-Foundation/Tonomy-ID/blob/master/prepare.app.config.ts and do this synchronously so if it fails the build fails